src/main.rs

```diff
@@ -1,198 +1,197 @@
-use aderyn::{
-    context::loader::ContextLoader,
-    framework::{
-        foundry::{load_foundry, read_foundry_output_file},
-        hardhat::load_hardhat,
-    },
-    read_file_to_string, run,
-    visitor::ast_visitor::Node,
-};
-use clap::Parser;
-use std::{fs::read_dir, path::PathBuf};
-use tokei::{Config, LanguageType};
+// use aderyn::{
+//     context::loader::ContextLoader,
+//     framework::{
+//         foundry::{load_foundry, read_foundry_output_file},
+//         hardhat::load_hardhat,
+//     },
+//     read_file_to_string, run,
+// };
+// use clap::Parser;
+// use std::{fs::read_dir, path::PathBuf};
+// use tokei::{Config, LanguageType};
 
-#[derive(Parser, Debug)]
-#[command(author, version, about, long_about = None)]
-struct Args {
-    /// Foundry or Hardhat project root directory
-    root: String,
-}
+// #[derive(Parser, Debug)]
+// #[command(author, version, about, long_about = None)]
+// struct Args {
+//     /// Foundry or Hardhat project root directory
+//     root: String,
+// }
 
-enum Framework {
-    Foundry,
-    Hardhat,
-}
+// enum Framework {
+//     Foundry,
+//     Hardhat,
+// }
 
 fn main() {
-    let args = Args::parse();
+    //     let args = Args::parse();
 
-    // Detect the framework
-    println!("Detecting framework...");
-    let root_path = PathBuf::from(&args.root);
-    let framework = detect_framework(root_path.clone()).unwrap_or_else(|| {
-        // Exit with a non-zero exit code
-        eprintln!("Error detecting framework");
-        std::process::exit(1);
-    });
+    //     // Detect the framework
+    //     println!("Detecting framework...");
+    //     let root_path = PathBuf::from(&args.root);
+    //     let framework = detect_framework(root_path.clone()).unwrap_or_else(|| {
+    //         // Exit with a non-zero exit code
+    //         eprintln!("Error detecting framework");
+    //         std::process::exit(1);
+    //     });
 
-    let src_path: String;
-    let mut context_loader = ContextLoader::default();
+    //     let src_path: String;
+    //     let mut context_loader = ContextLoader::default();
 
-    // This whole block loads the solidity files and ASTs into the context loader
-    // TODO: move much of this gutsy stuff into the foundry / hardhat modules.
-    match framework {
-        Framework::Foundry => {
```

Comment:
Change this whole flow to something more similar to solidity-rs loading from hardhat and foundry